tests/functional/adapter/test_model_grants.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from dbt.context.base import BaseContext  # diff_of_two_dicts only
+from dbt.tests.adapter.grants.test_model_grants import BaseModelGrants
+
+
+@pytest.mark.hive
+# TODO: setup roles in Galaxy
+@pytest.mark.skip_profile("starburst_galaxy")
```

Comment:
Can we test with starburst enterprise and also with trino-hive?

Reply:
We actually test with Hive's sql standard already. Galaxy and SEP will be tested later (see #146 and #147).